datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/dataset/mappers/DatasetMapper.java

```diff
@@ -110,6 +117,12 @@ public Dataset apply(@Nonnull final EntityResponse entityResponse) {
 
     private void mapDatasetKey(@Nonnull Dataset dataset, @Nonnull DataMap dataMap) {
         final DatasetKey gmsKey = new DatasetKey(dataMap);
+        System.out.println("~~~~~~~~~~~~~~~~~~~~~~NOW~~~~~~~~~~~~~~~");
+        System.out.println(gmsKey);
```

Comment:
sames

Reply:
this is what i get for rushing a PR up at the end of the day haha